<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Appearance</class>
 <widget class="QWidget" name="Appearance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="interface_tab">
      <attribute name="title">
       <string>Interface</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="animations_group">
         <property name="title">
          <string>Effects</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Animation Effects</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QGridLayout" name="gridLayout">
             <property name="margin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QCheckBox" name="menuCheckBox">
               <property name="text">
                <string>Open/Close Menu (Experimental)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="searchingCheckBox">
               <property name="text">
                <string>Searching</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="searchBarCheckBox">
               <property name="text">
                <string>SearchBar</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="tabsCheckBox">
               <property name="text">
                <string>Open/Close Tabs</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QCheckBox" name="toolbarCheckBox">
               <property name="text">
                <string>ToolBars</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Graphical Effects</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="gradiant_back_check">
               <property name="text">
                <string>Gradiant Background</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="colorized_menu_check">
               <property name="text">
                <string>Colorized Menu</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="widget_style_group">
         <property name="title">
          <string>Style</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="widget_style_label">
            <property name="text">
             <string>Widget Style</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="widget_style_combo">
            <property name="minimumSize">
             <size>
              <width>173</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="graphics_system_group">
         <property name="title">
          <string>Qt Graphics System</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="auto_check">
            <property name="text">
             <string>System Default</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="native_check">
            <property name="text">
             <string>Native</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="opengl_check">
            <property name="text">
             <string>OpenGL (Experimental)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="raster_check">
            <property name="text">
             <string>Raster</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="languagegroupBox">
         <property name="title">
          <string>Language Settings</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="languagelabel">
            <property name="text">
             <string>Select Language </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="languagecomboBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="color_tab">
      <attribute name="title">
       <string>Colors (Experimental)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QScrollArea" name="colors_area">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="colors_widget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>86</width>
            <height>16</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="colors_layout"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="icon_tab">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Icon Manager</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Icon Themes</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="margin">
           <number>1</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Enable Silicon Icon Themes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="iconList">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="addIconBtn">
           <property name="text">
            <string>Add New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeIconBtn">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="getIconBtn">
           <property name="text">
            <string>Get From Internet</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="themesTab">
      <attribute name="title">
       <string>Themes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="themeGroup">
         <property name="title">
          <string>Themes</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>1</number>
          </property>
          <item>
           <widget class="QListWidget" name="themeList"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="margin">
           <number>1</number>
          </property>
          <item>
           <widget class="QPushButton" name="addThemeBtn">
            <property name="text">
             <string>Add New</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeThemeBtn">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getThemeBtn">
            <property name="text">
             <string>Get From Internet</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
